app/views/Export/ExportSelectHA.js

```diff
@@ -18,24 +19,19 @@ import { Typography } from '../../components/Typography';
 import Colors from '../../constants/colors';
 import fontFamily from '../../constants/fonts';
 import { Theme } from '../../constants/themes';
-import { HCAService } from '../../services/HCAService';
-
-const getParsedAuthorities = async () => {
-  const authorities = await HCAService.getAuthoritiesList();
-  return authorities.map((authority) => ({
-    name: Object.keys(authority)[0],
-    url: Object.values(authority)[0][0].url,
-  }));
-};
+import getHealthcareAuthorities from '../../store/actions/healthcareAuthorities/getHealthcareAuthoritiesAction';
+import healthcareAuthorityOptionsSelector from '../../store/selectors/healthcareAuthorityOptionsSelector';
 
 export const ExportSelectHA = ({ navigation }) => {
   const { t } = useTranslation();
 
-  const [authorities, setAuthorities] = useState(null);
-  const [selectedAuthority, setSelectedAuthority] = useState(null);
+  const dispatch = useDispatch();
   useEffect(() => {
-    getParsedAuthorities().then(setAuthorities);
-  }, []);
+    dispatch(getHealthcareAuthorities());
+  }, [dispatch]);
+
+  const authorities = useSelector(healthcareAuthorityOptionsSelector);
```

Comment:
I normally advocate for inline selectors when not using reselect, but with the `ts` introduction I'm a fan of this selector because it adds more safety across the views in `js` and redux in `ts` separation. The ts validation that we're selecting real & accessible redux state is a big win for stability I think.